Add infos and format of our meetings

dragetd opened the pull request.

dragetd committed:
Add infos and format of our meetings

fes0j reviewed (commented):
Unter dem Stichpunkt Allgemein sollte genauer stehen, was mit "Thema anfügen" gemeint ist. Ich wünsche mir eine Ergänzung in der Form "Wir benutzen ein geteiltes Dokument das allen zugänglich ist. Darauf werden die Themen eingetragen. Während der Sitzung wird in daraus dann ein Protokoll."

Bei den Hackgruppen sollte vielleicht expliziter stehen, dass mehrere parallel stattfinden können.

Commit:
Corrections

Lazalatin commented:
Habe einfach mal ein paar kleinere Schreibfehler korrigiert.
Bin sonst zufrieden.

dragetd commented:
Vorschlag von Innay:

Kurzvorstellungen zu Themen die man evtl. in einer kleineren Runde besprechen will auch dann ohne viele Kommentare und am Anfang des Stammtisches, dann mit dem Verweis auf Hackgruppen/Diskussionsrunden am Ende

davidmehren commented on doc/meetings.md:
```diff
@@ -0,0 +1,15 @@
+
+## Protokoll: Aufteilung in drei Abschnitte
+  - Orga-Themen (Organisatorische Dinge der FOSS-AG und ihrer Projekte)
+  - Stammtisch (Vorstellung neuer und cooler Projekte/-ideen zu FOSS-Themen. Bevorzugt kleine Vorträge evtl. mit Folien. Alternativ, wenn man sich selbst damit nicht beschätigt hat, nur eine kurze Vorstellung mit Verweis auf eine Hackgruppe.)
```
`beschätigt` -> `beschäftigt`

davidmehren replied:
Würde ggf. den Nebensatz "wenn man sich selbst damit nicht beschätigt hat" komplett streichen. Ich kann mich ja auch mit was beschäftigt haben und dazu eine Hackgruppe haben wollen.

davidmehren commented on doc/meetings.md:
```diff
@@ -0,0 +1,15 @@
+## Protokoll: Aufteilung in drei Abschnitte
+  - Orga-Themen (Organisatorische Dinge der FOSS-AG und ihrer Projekte)
+  - Stammtisch (Vorstellung neuer und cooler Projekte/-ideen zu FOSS-Themen. Bevorzugt kleine Vorträge evtl. mit Folien. Alternativ, wenn man sich selbst damit nicht beschätigt hat, nur eine kurze Vorstellung mit Verweis auf eine Hackgruppe.)
+  - Hackgruppen (Gemeinsames beschäftigen mit bestehenden/neuen Projekten (z.B. die im Stammtisch vorgestellt wurden) in kleineren Gruppen)
```
`beschäftigen` -> `Beschäftigen`

InnayTool replied:
Also ich finde wir müssen dafür jetzt eigentlich nicht einen neuen Begriff einführen, sondern können sowas sagen wie: "mit Verweis auf eine der FOSS-AG Sitzung nachgestellte Disskussion".
Ich finde mehr Namen für Dinge führen unter Umständen eher zu mehr Verwirrung und es ist vermutlich für neue Menschen einfacher zu verstehen, wenn man es einfach Ausspricht was man meint.
Und ich sehe den Mehrwert eines neuen Wortes dafür noch nicht wirklich.

dragetd replied:
I agree. Lass uns den extra Namen streichen und einfach von Diskussionsgruppen reden. :)

Ich schaue mal, dass ich das bearbeiten kann.

davidmehren reviewed (changes requested):
Bisschen Tippfehler und ein inhaltlicher Kommentar.

dragetd committed:
Textanpassungen nach PR #23

Vermeidung unnötiger Spezial-Namen für die Gruppen. Verdeutlichung des Fokus
auf Dine die die AG betreffen. Beheben kleiner Tippfehler.

InnayTool commented:
Ich glaube das ist noch zu unklar, wann diese Diskussionen sein sollen.

Lazalatin commented on doc/meetings.md:
```diff
@@ -0,0 +1,23 @@
+  - Orga-Themen
+    - Organisatorische Dinge der FOSS-AG und ihrer Projekte
+  - Stammtisch
+    - Zunächst kurze Vorstellung von Projekten über die es anschließsenden Diskussionsgruppen gibt
```
[...] über die es "anschließende" Diskussionsgruppen gibt

Lazalatin replied:
Wer write access hat sollte Gebrauch von der Commenting-Funktion machen :P

Lazalatin reviewed (changes requested):
Correction

Lazalatin commented:
Ich würde gerne den Passus wieder herstellen, welcher den Kurzvortrag als bevorzugtes Medium hervorhebt... Wie sind da die Meinungen?

fes0j reviewed (approved):
Rechtschreibkorrektur immer gut. Inhaltliche Spezifizierungen am Besten mit neuen PR's.

Lazalatin commented:
> Rechtschreibkorrektur immer gut. Inhaltliche Spezifizierungen am Besten mit neuen PR's.

Finde ich gut :)

dragetd committed:
Vollständige Vermeidung eines Begriffs für die kleineren Gruppe im Anschluss
